sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryAvroUtilsTest.java

```diff
@@ -95,7 +99,9 @@ public void testConvertGenericRecordToTableRow() throws Exception {
       record.put("birthday", 5L);
       record.put("flighted", Boolean.TRUE);
       record.put("sound", soundByteBuffer);
-      record.put("anniversary", new Utf8("2000-01-01"));
+      record.put("anniversary_date", new Utf8("2000-01-01"));
+      record.put("anniversary_datetime", new String("2000-01-01 00:00:00.000005"));
```

Comment:
yes, I filed a bug. And, our code should just work once the bug is fixed on BQ side.